lib/req/steps.ex

```diff
@@ -1390,7 +1390,7 @@ defmodule Req.Steps do
 
   ## Options
 
-    * `:raw` - if set to `true`, disables response body decompression. Defaults to `false`.
+    * `:raw` - if set to `true`, disables response body decompression & decoding. Defaults to `false`.
```

Comment:
This is fine but wdyt about something even more explicit, along the lines of?

```suggestion
    * `:raw` - if set to `true`, disables response body decompression. Defaults to `false`.
    
      Note: `:raw` option also disables response body decoding in the `decode_body/1` step.
```

Reply:
Sounds great to me! I didn't wanna be as intrusive as I thought there may have been a reason for not mentioning it. Let me fix that up :)